packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -253,7 +260,7 @@ export const TypingIndicator = (props: TypingIndicatorProps): JSX.Element => {
 
   const typingUsersToRender = typingUsers.filter((typingUser) => typingUser.displayName !== undefined);
 
-  const indicatorComponent = getIndicatorComponent(
+  const indicatorComponent = IndicatorComponent(
```

Comment:
Please inline this definition on line 272.

Reply:
Any particular reason why?

Reply:
Any particular reason?

Reply:
I'm a bit wary of using Component functions outside of JSX syntax. It's all too easy to end up wrapping this in a `useEffect` or `useMemo` that React then complains about. We had this problem (have it still?) in `main` last week.